CMakeLists.txt

```diff
@@ -63,8 +63,10 @@ option(VELOX_ENABLE_S3 "Build S3 Connector" OFF)
 option(VELOX_ENABLE_HDFS "Build Hdfs Connector" OFF)
 option(VELOX_ENABLE_PARQUET "Enable Parquet support" OFF)
 option(VELOX_ENABLE_ARROW "Enable Arrow support" OFF)
+
 option(VELOX_BUILD_TEST_UTILS "Enable Velox test utilities" OFF)
 option(VELOX_BUILD_PYTHON_PACKAGE "Enable Velox Python bindings" OFF)
+option(VELOX_BUILD_BENCHMARKS "Builds Velox benchmarks" OFF)
```

Comment:
The options for VELOX_ENABLE_BENCHMARKS* also say 'Build Velox benchmakrs' maybe change them to enable. Also it would be nice if you can add a comment that the Enable* flags are for internal use.